src/routes/safe/components/Transactions/GatewayTransactions/modals/RejectTxModal.tsx

```diff
@@ -38,6 +38,7 @@ export const RejectTxModal = ({ isOpen, onClose, gwTransaction }: Props): React.
   const dispatch = useDispatch()
   const safeAddress = useSelector(safeParamAddressFromStateSelector)
   const classes = useStyles()
+  const [nonce] = useState(gwTransaction.executionInfo?.nonce ?? 0)
```

Comment:
why does it have to be a state variable?

Reply:
:point_up_2: I'm curious too

Reply:
tbh I'm not sure why I did that :facepalm:

Reply:
reverted